### PR TITLE
[TRANSLATION] Fix Simplified Chinese Translations

### PR DESCRIPTION
**Pull request by CNAmira**

### CODE REVIEW

**Comment by sanchaez** [comment]
Thank you for your contribution! Is this pull request still WIP?

**Comment by CNAmira** [comment]
Yes

**Comment by CNAmira** [comment]
It's okay to be merged I think.

File: base/applications/games/winmine/lang/zh-CN.rc [hunk 1]
```diff
@@ -2,7 +2,7 @@ LANGUAGE LANG_CHINESE, SUBLANG_CHINESE_SIMPLIFIED
 
 STRINGTABLE
 BEGIN
-    IDS_APPNAME "Wine地雷"
+    IDS_APPNAME "Wine 扫雷"
```
Comment (binarymaster) [inline base/applications/games/winmine/lang/zh-CN.rc]: Shouldn't these changes be sent upstream to Wine? :thinking:

Comment (CNAmira) [reply base/applications/games/winmine/lang/zh-CN.rc]: I'll contact with Wine's translator about this problem.

**Pull request merged**